incubator/spark/README.md

```diff
@@ -0,0 +1,103 @@
+# Apache Spark Helm Chart
+
+Apache Spark is a fast and general-purpose cluster computing system including Apache Zeppelin.
+
+* http://spark.apache.org/
+* https://zeppelin.apache.org/
```

Comment:
Im fine with leaving it in here for now. Once we have a better story on vendoring we can break it out.